content/6_Advanced/SPneg.mdx

```diff
@@ -214,7 +214,89 @@ int main() {
 ```
 
 </CPPSection>
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class CyclesFinding {
+	static final int N = 2501;
```

Comment:
make $n$ 2500 and add $1$ when you create your arrays

Reply:
Fix it